src/JJMasterData.Core/WebComponents/JJTextFile.cs

```diff
@@ -318,7 +318,7 @@ internal HtmlBuilder GetButtonGroupHtml()
     private JJLinkButton GetLinkButton(string filename)
     {
         var btn = new JJLinkButton();
-        btn.IconClass = IconHelper.GetClassName(IconType.CloudDownload);
+        btn.IconClass = IconType.CloudDownload.GetClassName();
```

Comment:
Witchcraft?

Reply:
@LucioPelinson  Extension method on the enum 🧙

Reply:
![image](https://user-images.githubusercontent.com/52143624/206335268-a1bcae6c-cd58-4350-afa9-9a39b8aa86b2.png)